### PR TITLE
print calendar closes abruptly in chrome

### PR DESCRIPTION
**Pull request by teryhill**
print calendar closes abruptly in chrome but works in FireFox @pri2si17-1997 thanks for finding this

### CODE REVIEW

File: interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day_print/outlook_ajax_template.html [hunk 1]
```diff
@@ -511,7 +511,7 @@
             $patientid = $event['pid'];
             $commapos = strpos($event['patient_name'], ",");
             $lname = substr($event['patient_name'], 0, $commapos);
-	    $fname = substr($event['patient_name'], $commapos + 2);
+        $fname = substr($event['patient_name'], $commapos + 2);
```
Comment (aethelwulffe) [inline interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day_print/outlook_ajax_template.html]: Pleeze stop beeting up whitespace. :)  Maykes ztuff herd tew reed an fine reel chaynjuz.

File: interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day_print/outlook_ajax_template.html [hunk 2]
```diff
@@ -697,7 +697,7 @@
 <script>
 $(document).ready(function(){
     var win = top.printLogPrint ? top : opener.top;
-    if (win.printLogPrint(window)) window.close();
+    win.printLogPrint(window);
```
Comment (aethelwulffe) [inline interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day_print/outlook_ajax_template.html]: Firefox still works after this?

File: interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day_print/outlook_ajax_template.html (hunk 1 above)
Comment (teryhill) [inline interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day_print/outlook_ajax_template.html]: Its the editor

File: interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day_print/outlook_ajax_template.html (hunk 2 above)
Comment (pri2si17-1997) [inline interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day_print/outlook_ajax_template.html]: Yes @aethelwulffe

**Comment by aethelwulffe** [comment]
Recommend you look at settings for whitespace in you editor, and if you use gitHUB for windows, there are tools there that help with that stuff too.

File: interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day_print/outlook_ajax_template.html (hunk 2 above)
Comment (teryhill) [inline interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day_print/outlook_ajax_template.html]: yes I tested it and it works

**Comment by aethelwulffe** [comment]
Merged this one liner.  We had three sets of eyes, and two independent tests.

**Pull request merged**